rollup.config.js

```diff
@@ -5,6 +5,7 @@ import livereload from 'rollup-plugin-livereload';
 import { terser } from 'rollup-plugin-terser';
 import json from 'rollup-plugin-json';
 import replace from '@rollup/plugin-replace';
+import postcss from 'rollup-plugin-postcss';
```

Comment:
why is this needed?

Reply:
I can't say I'm great with Rollup, but I get this error when the postcss plugin is excluded.

Error: Unexpected token (Note that you need plugins to import files that are not JavaScript)

Reply:
Yes, this is the preferred way to be able to run `import "@graph-paper/core/style.css` and other stylesheets, which is what happens when we run `import { Button } from "@graph-paper/button`.